test/helpers/test.go

```diff
@@ -832,17 +938,50 @@ func (t *TestOptions) ExpectThatIssueIsAssignedToUser(provider gits.GitProvider,
 	return RetryExponentialBackoff(TimeoutProwActionWait, f)
 }
 
-// AttemptToLGTMOwnPR return an error if the /lgtm fails to add the lgtm label to PR
-func (t *TestOptions) AttemptToLGTMOwnPR(provider gits.GitProvider, owner string, repo string) error {
+// MostRecentOpenPullRequestForOwnerAndRepo returns the most recently opened pull request for a given owner/repo. If
+// there aren't any open PRs, it will return nil.
+func (t *TestOptions) MostRecentOpenPullRequestForOwnerAndRepo(provider gits.GitProvider, owner string, repo string) (*gits.GitPullRequest, error) {
```

Comment:
why the most recent? Do we not know the id of the PR? Can we not explicitly get the right PR? This is imo fragile.

Reply:
Because I haven’t refactored the existing ChatOps tests yet, basically, so I just emulated their preexisting behavior while moving the common bits into a provider-driven function. Within the next couple days, I’ll be redoing the ChatOps tests to be saner and get rid of this function.